### ISSUE TITLE
afterCreate no longer contains id of created item

### ISSUE THREAD

**Issue by dustinbolton** [balderdashy/sails-mysql#233]
I reported this issue ( https://github.com/balderdashy/waterline/issues/661 ) to the waterline project & others noted they have the same issue.  However I was told to report the issue to the individual adapter's project.  The work-around mentioned in the reported issue no longer works.  I am currently running sails-mysql v0.10.11. The ID not being returned in afterCreate has the config (which used to work around this issue but no longer does):
```
id: {
	type: 'integer',
	autoIncrement: true,
	primaryKey: true
}
```

**Comment by sailsbot** [comment]
Thanks for posting, @dustinbolton.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
